docs/DRIVE_PERMISSION_CHECK.md

```
@@ -21,13 +21,15 @@
 ### ステップ1: サービスアカウントのメールアドレスを確認
 
 **GCP Console（Web UI）で確認**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)